projects/storefrontstyles/scss/components/product/list/_active-facets.scss

```diff
@@ -15,6 +15,17 @@
     }
   }
 
+  h2 {
```

Comment:
We can leave a comment to remove the `h4` tag from styles too and include it in our migration docs.